libraries/engage/product/components/Swatch/Swatch.jsx

```diff
@@ -7,19 +7,18 @@ import { css } from 'glamor';
 import { PRODUCT_SWATCH } from '@shopgate/pwa-common-commerce/product/constants/Portals';
 import { SurroundPortals } from '../../../components';
 import { isBeta, useWidgetStyles } from '../../../core';
+import { WIDGET_ID } from './constants';
 import { SwatchColor } from './SwatchColor';
 import { SwatchTexture } from './SwatchTexture';
 import { swatchGrid, itemSelected } from './style';
 
-const widgetId = '@shopgate/engage/product/Swatch';
-
 /**
  * The swatch component.
  * @param {Object} props The component props.
  * @returns {JSX}
  */
 const SwatchUnwrapped = ({
-  testId, maxItemCount, swatch, widgetPath, onClick,
+  testId, maxItemCount, swatch, widgetId, onClick,
```

Comment:
A widget is defined by it's id. If we keep this up, things can get complicated, because the widget suddenly doesn't know about it's own type, because it's somehow passed in from outside.
When we introduce the automated renderer this will break.